src/domains/sign/io_sign_issuer_func.tf

```diff
@@ -22,6 +22,7 @@ locals {
       IoServicesSubscriptionKey                       = module.key_vault_secrets.values["IOApiSubscriptionKey"].value
       PdvTokenizerApiBasePath                         = "https://api.uat.tokenizer.pdv.pagopa.it"
       PdvTokenizerApiKey                              = module.key_vault_secrets.values["TokenizerApiSubscriptionKey"].value
+      EventHubsConnectionString                       = module.key_vault_secrets.values["EventHubsConnectionString"].value
```

Comment:
Renamed to `BillingEventHubConnectionString` and added `AnalyticsEventHubConnectionString` https://github.com/pagopa/io-sign/pull/73

Reply:
Done 👍